src/main/java/de/muenchen/dave/domain/mapper/detektor/MessstelleCronMapper.java

```diff
@@ -0,0 +1,59 @@
+package de.muenchen.dave.domain.mapper.detektor;
```

Comment:
Der exisitiert nicht mehr im Sprint. Ich sollte den umbenennen in **MessstelleReceiverMapper **.

Wieder entfernen

Reply:
done